tests/test_copy_novaseqx_flow_cell.py

```diff
@@ -0,0 +1,180 @@
+from pathlib import Path
+import pytest
+from typing import Set
+from cg.cli.demultiplex.copy_novaseqx_data import get_latest_analysis_directory
+
+from cg.cli.demultiplex.demux import (
+    copy_flow_cell_analysis_data,
+    is_ready_for_post_processing,
+)
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+@pytest.fixture
+def latest_analysis_version() -> str:
+    return "2"
+
+
+@pytest.fixture
+def flow_cell_name() -> str:
+    return "20230427_LH00188_0001_B223YYCLT3"
```

Comment:
```suggestion
@pytest.fixture(name="novaseq_x_flow_cell_directory_name", scope= "session")
def fixture_novaseq_x_flow_cell_directory_name() -> str:
"""Returns the directory name of a novaseqX flow cell."""
    return "20230427_LH00188_0001_B223YYCLT3"
```

Reply:
Skipping the fixture prefix and the doc string in this case since I think it is not DRY

Reply:
Updated fixture names to be more specific and moved them to the root conftest